### PR TITLE
Added the 4 logos ✅

### PR DESCRIPTION
**Pull request by lavakush07**
Signed-off-by: Lavakush biyani <70131581+lavakush07@users.noreply.github.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 2a3e8a688e019b7a5e1f312c84bd1bf96c8f9e0a at: https://631d69f120301d5889bc0bb1--layer5.netlify.app

File: src/sections/Community/Handbook/repo-data.js [hunk 1]
```diff
@@ -301,6 +304,7 @@ export const repo_data = [
       },
       {
         project: "Service Mesh Patterns Action",
+        image: patterns,
```
Comment (legendarykamal) [inline src/sections/Community/Handbook/repo-data.js:307]: image

@leecalcote shouldn't we use  github actions here ?

File: src/sections/Community/Handbook/repo-data.js [hunk 2]
```diff
@@ -274,12 +275,14 @@ export const repo_data = [
...
       {
         project: "SMP Action",
+        image: github,
         language: "shell",
         description: "GitHub Action to run SMP Performance Benchmarks on CI/CD pipelines.",
         repository: "https://github.com/layer5io/meshery-smp-action"
       },
       {
         project: "Meshery-Perf",
+        image:github,
```
Comment (legendarykamal) [inline src/sections/Community/Handbook/repo-data.js:285]: and meshery logo here ? :thinking:

**Review by legendarykamal — CHANGES_REQUESTED** [review]
1> Confirmation needed in these 2 places ! 

2> @lavakush07 can you find a way to flip these images with the dark one so that that get aligned with other similar ones . See below 

image

File: src/sections/Community/Handbook/repo-data.js (hunk 2 above)
Comment (leecalcote) [reply src/sections/Community/Handbook/repo-data.js:285]: Right. Yes, Meshery logo here.

File: src/sections/Community/Handbook/repo-data.js (hunk 1 above)
Comment (leecalcote) [reply src/sections/Community/Handbook/repo-data.js:307]: To be consistent in presentation with other GitHub Action projects that we have - yes, you’re right.

**Comment by legendarykamal** [comment]
@lavakush07 please do the needful .

File: src/sections/Community/Handbook/repo-data.js (hunk 2 above)
Comment (leecalcote) [reply src/sections/Community/Handbook/repo-data.js:285]:
```suggestion
        image:meshery,
```

File: src/sections/Community/Handbook/repo-data.js (hunk 1 above)
Comment (leecalcote) [reply src/sections/Community/Handbook/repo-data.js:307]: Although, I really don't mind having the Service Mesh Patterns logo used a bit more than it is. 😀

**Review by leecalcote — APPROVED** [review]

**Pull request merged**